src/mon/PGMap.cc

```diff
@@ -3568,13 +3648,24 @@ int process_pg_map_command(
 
     pg_map.get_filtered_pg_stats(state, pool, osd, primary, pgs);
 
-    if (f && !pgs.empty()) {
-      pg_map.dump_filtered_pg_stats(f, pgs);
-      f->flush(*odata);
-    } else if (!pgs.empty()) {
-      pg_map.dump_filtered_pg_stats(ds, pgs);
-      odata->append(ds);
-      odata->append(omap_stats_note);
+    if (!osd_df_by_pool) {
```

Comment:
i think new status-reporting commands should be implemented as mgr modules. so this command should be added to https://github.com/ceph/ceph/blob/e487604ad2af483a54e4616aff2f313a266d99d3/src/pybind/mgr/status/module.py#L17

@neha-ojha @liewegas what do you think?

Reply:
@tchaikov  I implement this command in mgr modules.